File: URI/Beginner/1044.cpp
#include <iostream>
#include <cstdio>
#include <cstdlib>
#include <string>
#include <math.h>

using namespace std;

int main()
{
    int a, b;
    cin >> a >> b;

    if (a >= b)
    {
        if (a % b == 0)
        {
            cout << "Sao Multiplos" << endl;
        }
        else
        {
            cout << "Nao sao Multiplos" << endl;
        }
    }
    else if (b > a)
    {
        if (b % a == 0)
        {
            cout << "Sao Multiplos" << endl;
        }
        else
        {
            cout << "Nao sao Multiplos" << endl;
        }
    }

    return 0;
}

File: Python/URI/Beginner/1044.py
import sys

a, b = input().split()

a = int(a)
b = int(b)

if ((a > b) and ((a%b)==0)) or ((b > a) and ((b%a)==0)):
	print ("Sao Multiplos")
else:
	print ("Nao sao Multiplos")